ignite/engine/engine.py

```diff
@@ -298,7 +298,12 @@ def execute_something():
 
         self._assert_allowed_event(event_name)
 
-        event_args = (Exception(),) if event_name == Events.EXCEPTION_RAISED else ()
+        event_args = ()  # type: Tuple[Any, ...]
+        if event_name == Events.EXCEPTION_RAISED:
+            event_args += (Exception(),)
+        elif event_name == Events.TERMINATE_SINGLE_EPOCH:
```

Comment:
What is the engine supposed to do after receiving  `terminate_single_epoch` command?